package taocipian.sample;

option cc_generic_services = true;

message EchoRequest {
	required string message = 1;
}

message EchoResponse {
	required string message = 1;
}

service MasterServer {
	rpc Echo(EchoRequest) returns(EchoResponse);
}
